frontend/src/components/collaborator/CollaboratorEdit.vue

```diff
@@ -152,14 +153,7 @@ export default {
     return {
       resendingEmail: false,
       emailSent: false,
-      entityProperties: [
-        'abbreviation',
-        'camp',
-        'color',
-        'inviteEmail',
-        'role',
-        'status',
-      ],
+      entityProperties: ['abbreviation', 'color', 'role', 'status'],
```

Comment:
why could you remove the entityProperties camp and inviteEmail here?

Reply:
Because they are not needed and can't be changed here.